### PR TITLE
Chore/cleanup 2

### PR DESCRIPTION
**Pull request by manuelbieh**
Some more cleanup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
     "classnames": "^2.2.6",
     "clipboard-copy": "^3.1.0",
     "humps": "^2.0.1",
+    "normalize.css": "^8.0.1",
```
Comment (drinchev) [inline package.json]: Hmm why do we need this btw?

Comment (manuelbieh) [reply package.json]: I don't know, to be honest. I removed it but then the build failed, so I added it back 🤷‍♂  

```
Module not found: Can't resolve '~/htdocs/osram/platform/react/node_modules/normalize.css' in '~/htdocs/osram/platform/react'
```

Comment (manuelbieh) [reply package.json]: Ah, here is the reason:
https://github.com/lightelligence-io/react/blob/alpha/styleguide.config.js#L170

Comment (drinchev) [reply package.json]: Yeah but the styleguide uses that, so it's a devDependency.

**Comment by drinchev** [comment]
@manuelbieh let's merge this when you change the dependency.

**Pull request merged**